src/functions/formattedEnum.ts

```diff
@@ -0,0 +1,70 @@
+import { AssertionError } from 'assert';
+import { isArray } from 'lodash';
+import { IFunction, IFunctionResult, IRule, RuleFunction } from '../types';
+import { schema } from './schema';
+
+export interface IFormattedEnumOptions {
+  reportingThreshold: number;
+}
+
+export type FormattedEnumRule = IRule<RuleFunction.FORMATTED_ENUM, IFormattedEnumOptions>;
+
+export const formattedEnum: IFunction<IFormattedEnumOptions> = (
+  targetVal,
+  opts,
+  paths,
+  otherValues,
+): void | IFunctionResult[] => {
+  assertValidOptions(opts);
+
+  const { enum: enumValues, ...innerSchema } = targetVal;
+
+  if (!isArray(enumValues)) {
```

Comment:
```suggestion
  if (!Array.isArray(enumValues)) {
```
let's use native :D

Reply:
Fixed. You're right.

FWIW, I don't even remember why I switched over to lodash... :-/